### PR TITLE
Feat cofounder idea detail page

### PR DESCRIPTION
**Pull request by Emnetdegafe**

### CODE REVIEW

File: src/components/Cofounder/BidSlider.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    //             }
+    //         })
+    //         .catch(err => {
+    //             if (err.status === 400) {
+    //             } else {
+    //                 console.error(err);
+    //             }
+    //         });
+
+    // }
```
Comment (DannyS-stack) [inline src/components/Cofounder/BidSlider.js:105]: remove this comment if we won't need it

File: src/App.js [hunk 2]
```diff
@@ -717,6 +718,22 @@ class App extends Component {
                     />
                   );
                 }}
+              />
+                            <Route
```
Comment (DannyS-stack) [inline src/App.js:722]: this needs some backspaces

**Review by DannyS-stack — CHANGES_REQUESTED** [review]
in app.js line 722 the Route has strange formatting that could be easily fixed
in cofounder/bidslider there is a large comment line 84-105 that needs to be removed.

**Review by DannyS-stack — APPROVED** [review]

**Review by Kruxk — APPROVED** [review]
LGTM!

**Pull request merged**